.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
+  release:
+    types: [published, created, edited]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
needs some more research here...  I'm not sure "ubuntu-latest" contains the podman binary for running the make image-* targets.
and I don't see an "actions/podman" or similar available.
@jomkz @bcrochet please hold on. don't review yet.

Reply:
@komish maybe you can help?
I need either a Runner for "runs-on" which contains a podman binary,  or the podman variant of docker/build-push-action@v2

Reply:
Thanks @komish   looks like the ubuntu runner has both docker and podman 
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md